packages/forms/frontend/sirius-components-widget-reference/src/components/ValuedReferenceAutocomplete.tsx

```diff
@@ -72,6 +98,45 @@ export const ValuedReferenceAutocomplete = ({
   };
   const classes = useStyles(props);
 
+  const { addErrorMessage } = useMultiToast();
+  const [open, setOpen] = useState<boolean>(false);
+  const [options, setOptions] = useState<GQLReferenceValue[] | null>(null);
```

Comment:
Use one state to help preventing invalid combined states

Reply:
Done